test/api/connectors/ConnectorSpec.scala

```diff
@@ -174,8 +174,7 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
     MockedAppConfig.desEnvironment returns "des-environment"
     MockedAppConfig.desEnvironmentHeaders returns Some(allowedDesHeaders)
 
-    MockedAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> false)
-
+    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(true).anyNumberOfTimes()
```

Comment:
Was false before; should be false now:
```suggestion
    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(false).anyNumberOfTimes()
```

Reply:
Good catch.